src/sources/exec/mod.rs

```diff
@@ -250,13 +262,49 @@ impl SourceConfig for ExecConfig {
                     decoder,
                     cx.shutdown,
                     cx.out,
+                    log_namespace,
                 )))
             }
         }
     }
 
-    fn outputs(&self, _global_log_namespace: LogNamespace) -> Vec<Output> {
-        vec![Output::default(self.decoding.output_type())]
+    fn outputs(&self, global_log_namespace: LogNamespace) -> Vec<Output> {
+        let log_namespace = global_log_namespace.merge(Some(self.log_namespace()));
```

Comment:
```suggestion
        let log_namespace = global_log_namespace.merge(log_namespace());
```

I'd drop the function above and just do this?